submissions/DmytryjK/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli Popup</title>
+    <link rel="icon" type="image/x-icon" href="images/favicon.png">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <a class="header__logo-link" href="/">
+                <img class="header__logo-img" src="images/logo.svg" alt="site logotype">
+            </a>
+            <nav class="header__navigation">
+                <ul class="header__list">
+                    <li class="header__items">
+                        <a class="header__links" href="#">Mail</a>
+                    </li>
+                    <li class="header__items">
+                        <a class="header__links" href="#">Images</a>
+                    </li>
+                    <li class="header__items">
+                        <label class="header__popup-label" for="checkbox">
```

Comment:
When user opens a popup using keyboard and press the `Tab` key focus must go to the first interactive element inside the popup. In your case focus goes to the popup wrapper and after second click goes to the the Calendar icon.

Reply:
I try to use somesing like this:
```
input:checked ~ .header__popup-submenu > li:first-child > a:active  {
            outline: solid $color-state-first; 
        }
```
or 

```
input:focus ~ .header__popup-submenu > li:first-child > a:active  {
            outline: solid $color-state-first; 
        }
```
But it doesn't working :( :

Could you show me the way in which I should to move for solve this task?

Reply:
When checkbox gets focus you highlight the label, what you put inside the label?)

Reply:
My list items with links. Don't have any hypothesis. 
Maybe my structure isn't correctly?

```
<label>
     <input>
     ul>li>a...
</label>
```

Do i have to do something like this? 
```
<label></label>
<input>
ul>li>a...
```

This code change outline in the links icons, when i click on menu button, but won't change focus((

```
input:focus ~ .header__popup-submenu > li:first-child > a {
            outline: solid $color-state-first;
            border-radius: 5px;
} 
```

Reply:
You need something like this:
```HTML
<input type="checkbox" >
<label></label>
<div class="popup></div>
```
In this case, when checkbox will be in focus, you can highlight the label and show popup)

Reply:
Good night, sorry, but i really don't know, how to change focus from checked input to first link in navbar menu. I tried to find some information about this case, but didn't find anything :( Can we check this in discord?)

Reply:
@DmytryjK I have already given you the correct answer to this question) Look carefully at the code example and compare it with your code)

Reply:
I tried rewrite my code as in the example above. But it was the same story: focus remains on the menu button element. Honestly I don't understand what the different between:
```
<label>
 <input></input>
 <ul>
  <li></li>
 </ul>
</label>
```

and

```
<label></label>
<input></input>
<ul>
  <li></li>
</ul>
```

In both cases, when I press the menu button, submenu opens without problems. But the focus doesn't jump to the 1st element from the list.
I don't understand the logic of how exactly to change the target when clicking on the checkbox of the menu button.

Reply:
Thanks for call. Issues have been resolved. Need clear cache in browser for correct review DEMO site.